packages/framework/src/Model/Customer/User/Role/CustomerUserRoleProvider.php

```diff
@@ -42,4 +45,16 @@ public function canCustomerUserSeePrices(?CustomerUser $customerUser): bool
 
         return in_array(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES, $roles, true);
     }
+
+    /**
+     * @return bool
+     */
+    public function canCurrentCustomerUserSeePrices(): bool
```

Comment:
with this method, should the class still be named `CustomerUserRoleProvider`? I would rather name it facade or resolver, however, I see there is already `canCustomerUserSeePrices` method so I can probably live with the current name :slightly_smiling_face:

Reply:
I like name `CustomerUserRoleProvider`, but commit was made in https://github.com/shopsys/shopsys/pull/3356/commits/5e085a7152286289044c90bbb2ba70258a3e59e7, branch that is build on top of this one

Is it ok ? Or should I remove it ?